pyap_beauhurst/address.py

```diff
@@ -11,17 +9,14 @@
     :license: MIT, see LICENSE for more details.
 """
 
-from .packages import six
-
-
-class Address(object):
 
+class Address:
     def __init__(self, **args):
         keys = []
         vals = []
-        for k, v in six.iteritems(args):
+        for k, v in args.items():
```

Comment:
I think `args` should typically be reserved for positional arguments.  These look more like `kwargs`?

Reply:
66dd467 :+1: